als-server/shared/src/main/scala/org/mulesoft/als/server/modules/diagnostic/CleanDiagnosticTreeManager.scala

```diff
@@ -66,7 +66,7 @@ class CleanDiagnosticTreeManager(telemetryProvider: TelemetryProvider,
     val helper     = environmentProvider.amfConfiguration.modelBuilder()
     val refinedUri = uri.toAmfDecodedUri(environmentProvider.platform)
     helper
-      .parse(refinedUri, environmentProvider.environmentSnapshot())
+      .parse(refinedUri, environmentProvider.environmentSnapshot(), None)
```

Comment:
This might be a problem in the future when clean diagnostic requires the rulesets and semantic extensions

Reply:
los parse cambian todos de lugar con AMF 5 igual, problema para el lean del futuro que lo adopte, espero que este bien testeado :P

Reply:
Si, lo sospechaba a eso jajaja